src/constants.ts

```diff
@@ -16,6 +16,8 @@ const schema = z.object({
     .regex(/^\d\d:\d\d$/)
     .default('17:00'),
   OPENAI_API_KEY: z.optional(z.string()),
+  RUNN_API_BASE_URL: z.string(),
+  RUNN_API_TOKEN: z.string(),
```

Comment:
TODO: I need to get an API token for production and add it to the .env file of the production environment